src/python/WMCore/JobSplitting/EventAwareLumiByWork.py

```diff
@@ -67,8 +66,7 @@ def algorithm(self, *args, **kwargs):
         avgEventsPerJob = int(kwargs.get('events_per_job', 5000))
         lumiEventLimit = int(kwargs.get('max_events_per_lumi', 20000))
         totalEventLimit = int(kwargs.get('total_events', 0))
-        splitOnFile = bool(kwargs.get('halt_job_on_file_boundaries', True))
-        ignoreACDC = bool(kwargs.get('ignore_acdc_except', False))
+        splitOnFile = bool(kwargs.get('halt_job_on_file_boundaries', False))
```

Comment:
This is parameters for splitting so it can always set to different values in assignment stage, right?